database/buildSchema/28_review_assignment.sql

```diff
@@ -42,13 +45,36 @@ CREATE TABLE public.review_assignment (
     is_self_assignable boolean DEFAULT FALSE
 );
 
--- TRIGGER (Listener) on review_assignment table
+-- FUNCTION
+CREATE OR REPLACE FUNCTION public.unassign_review_without_sections ()
+    RETURNS TRIGGER
+        AS $review_assignment_event$
+    BEGIN
+        UPDATE
+            public.review_assignment
+        SET
+            status = 'AVAILABLE'
+        WHERE
+            id = NEW.id;
+        RETURN NULL;
+    END;
+$review_assignment_event$
+LANGUAGE plpgsql;
+
+-- TRIGGER (Listener) on review_assignment table: To update trigger
 CREATE TRIGGER review_assignment_trigger
     AFTER INSERT OR UPDATE OF trigger ON public.review_assignment
     FOR EACH ROW
     WHEN (NEW.trigger IS NOT NULL AND NEW.trigger <> 'PROCESSING' AND NEW.trigger <> 'ERROR')
     EXECUTE FUNCTION public.add_event_to_trigger_queue ();
 
+-- TRIGGER (Listener) on review_assignment table: To set status accordingly to assignedSections
+CREATE TRIGGER review_assignment_trigger2
+    AFTER UPDATE OF trigger ON public.review_assignment
```

Comment:
```suggestion
    AFTER UPDATE OF assigned_sections ON public.review_assignment
```

Wrong target column -- this fixes it.